image_classification/tensorflow/official/utils/arg_parsers/parsers.py

```diff
@@ -160,6 +160,27 @@ def __init__(self, add_help=False, data_dir=True, model_dir=True,
           metavar="<BS>"
       )
 
+    if enable_lars:
+      self.add_argument(
+         "--enable_lars", "-el", type=Boolean, default=False,
+         help='Enable LARS optimizer for large batch training.',
+         metavar="<EL>"
+      )
+
+    if label_smoothing:
+      self.add_argument(
+         "--label_smoothing", "-ls", type=float, default=0.0,
```

Comment:
Option string `-ls` is already defined for `--loss_scale`.

Reply:
done